packages/open-next/src/build.ts

```diff
@@ -489,6 +489,25 @@ async function createServerBundle(monorepoRoot: string) {
   injectMiddlewareGeolocation(outputPath, packagePath);
   removeCachedPages(outputPath, packagePath);
   addCacheHandler(outputPath);
+
+  if (options.minify) {
+    removeNodeModule(path.join(outputPath, "node_modules"), [
+      "@esbuild",
```

Comment:
my logs:
<img width="821" alt="image" src="https://github.com/sst/open-next/assets/43549299/e6ed0097-2d61-4b9c-89d5-0244b7f3683a">
can we remove some of them also?

Reply:
Very likely yes, but you'll need to test for yourself. You can probably remove : 
- `swc core`
- `sharp`
- `esbuild`
- `caniuse lite`
- `uglify-js`
- `terser`